File: uoj_12577_hajj-e-akbar.cpp
#include <iostream>
#include <string>

using namespace std;

int main()
{
  int counter = 1;
  string s;

  while ( cin >> s && s != "*")
    {
	if (s == "Hajj")
	  cout << "Case " << counter++ << ": Hajj-e-Akbar" << endl;
	else // "Umrah":
	    cout << "Case " << counter++ << ": Hajj-e-Asghar" << endl;
	  
    }
}

File: uoj_12577_hajj-e-akbar.py
# Tedious

counter = 1

while True:
    try: 
        s = input()
    except EOFError:
        break

    match s:
        case "*":
            break
        case "Hajj":
            print(f"Case {counter}: Hajj-e-Akbar")
        case "Umrah":
            print(f"Case {counter}: Hajj-e-Asghar")
        
    counter +=1

    
    
